### PR TITLE
Add French helpticket translations

### PR DESCRIPTION
**Pull request by KennedyLFC**
per https://www.smogon.com/forums/threads/rules-commands-translations.3645723/post-9682811

### CODE REVIEW

File: translations/french/helptickets.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+		"If you are a room driver or up in a public room, and you need help watching the chat, one or more global staff members would be happy to assist you!": "Si tu es room driver ou plus haut dans une room publique et que tu as besoin d'aide pour surveiller la room, un ou plusieurs membres du global staff seraient heureux de t'aider !",
+		"If your issue is not handled above, click the button below to talk to a global staff member. Please be ready to explain the situation.": "Si le problème rencontré n'est pas pris en compte par un des cas ci-dessus, clique sur le bouton ci-dessous pour discuter avec un membre du global staff. Merci de bien expliquer la situation.",
+		"Malformed help request.": "Requête mal formulée.",
+		"PM Harassment": "Harcèlement en messages privés",
+		"Battle Harassment": "Harcèlement en match",
+		"Inappropriate Username": "Pseudo inapproprié",
+		"Inappropriate Pokemon Nicknames": "Surnom de Pokémon inapproprié",
+		"Appeal": "Appel",
+		"IP-Appeal": "Appel IP",
+		"ISP-Appeal": "NOT SURE TBH LOL",
```
Comment (AnnikaCodes) [inline translations/french/helptickets.ts:48]: Please don't put this in the final translation. It's better to leave something in English than to totally obscure its meaning.

**Review by AnnikaCodes — CHANGES_REQUESTED** [review]

File: translations/french/helptickets.ts (hunk 1 above)
Comment (KennedyLFC) [reply translations/french/helptickets.ts:48]: My bad, I didn't notice this when scanning, I'll let the person who did the translation to know for future reference though

Comment (KennedyLFC) [reply translations/french/helptickets.ts:48]: Apparently he wants to review the rest of the translations too so I'll fix this along with any others he decides to alter

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**